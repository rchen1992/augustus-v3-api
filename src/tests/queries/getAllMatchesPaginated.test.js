const mockMatchesRepo = require('@tests/mockRepos/match');
const { testQuery } = require('@tests/utils');

testQuery({
    id: 'Get all matches paginated with limit and offset',
    query: `
        {
            matches(offset: 1, limit: 3) {
                id
                createdAt
            }
        }
    `,
    variables: {},
    context: () => {
        return {
            repos: {
                match: mockMatchesRepo,
            },
        };
    },
    expected: undefined,
});
