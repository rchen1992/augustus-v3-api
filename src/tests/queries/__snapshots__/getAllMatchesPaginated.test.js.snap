// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`query: Get all matches paginated with limit and offset 1`] = `
Object {
  "data": Object {
    "matches": Array [
      Object {
        "createdAt": "2018-12-07 05:46:42",
        "id": "1",
      },
      Object {
        "createdAt": "2018-09-17 01:38:13",
        "id": "4",
      },
      Object {
        "createdAt": "2018-08-22 00:10:00",
        "id": "2",
      },
    ],
  },
}
`;
